data/data-pipeline/data_pipeline/score/score_l.py

```diff
@@ -93,9 +93,37 @@ def _combine_island_areas_with_states_and_set_thresholds(
 
         return df, threshold_column_name
 
+    def _create_low_income_threshold(self, df: pd.DataFrame) -> pd.Series:
+        """
+        Returns a pandas series (really a numpy array)
+        of booleans based on the condition of the FPL at 200%
+        is at or more than some established threshold
+        """
+        return (
+            df[
+                field_names.POVERTY_LESS_THAN_200_FPL_FIELD
+                + field_names.PERCENTILE_FIELD_SUFFIX
+            ]
+            >= self.LOW_INCOME_THRESHOLD
+        )
+
+    def _increment_total_eligibility_exceeded(
+        self, columns_for_subset: list
+    ) -> None:
+        """
+        Increments the total eligible factors for a given tract
+        """
+
+        self.df[field_names.THRESHOLD_COUNT] += self.df[columns_for_subset].sum(
+            axis=1
+        )
+
     def add_columns(self) -> pd.DataFrame:
         logger.info("Adding Score L")
 
+        self.df[field_names.FPL_200_SERIES] = self._create_low_income_threshold(
+            self.df
+        )
```

Comment:
need to add self.df[field_names.THRESHOLD_COUNT] = 0 here

Reply:
wow....